library/events.go

```diff
@@ -66,6 +66,8 @@ func (e *Events) Allowed(event, action string) bool {
 		allowed = e.GetComment().GetEdited()
 	case constants.EventDeploy:
 		allowed = e.GetDeployment().GetCreated()
```

Comment:
```suggestion

```

We can ditch this case if we're setting the `EventAction` in `processDeploymentEvent` in the server